src/analytics/searchParamsEvent.ts

```diff
@@ -0,0 +1,57 @@
+import { AnalyticsEvent } from "./analyticsEvents";
+import { trackCustomEvent } from "./customEventTracker";
+
+/**
+ * Parameter mapping from URL shortcodes to meaningful names
+ */
+const URL_PARAM_MAPPING = new Map([
```

Comment:
No need to create this as it is duplicate of info use the paramReverseLookup in the UrlUtils.

In fact your approach may result in too many place that needs to change and track, is it easy to track the parameter in the rest api call level like searchReducer.tsx and inside all the searchResult functions. Track it before the call to axios.get () ?